src/tokenize_cleanup.cpp

```diff
@@ -1068,3 +1055,73 @@ static void check_template(chunk_t *start)
            (end != NULL) ? get_token_name(end->type) : "<null>");
    set_chunk_type(start, CT_COMPARE);
 } // check_template
+
+
+void cleanup_objc_property_with_parens(chunk_t *start)
+{
+   chunk_t *next = chunk_get_next_ncnl(start);
+
+   set_chunk_parent(next, start->type);
+
+   chunk_t *tmp = chunk_get_next_type(start, CT_PAREN_CLOSE, start->level);
+   if (tmp != NULL)
+   {
+      set_chunk_parent(tmp, start->type);
+      tmp = chunk_get_next_ncnl(tmp);
+      if (tmp != NULL)
+      {
+         chunk_flags_set(tmp, PCF_STMT_START | PCF_EXPR_START);
+
+         tmp = chunk_get_next_type(tmp, CT_SEMICOLON, start->level);
+         if (tmp != NULL)
+         {
+            set_chunk_parent(tmp, start->type);
+         }
+      }
+   }
+   mark_selectors_in_property(start);
+   mark_attributes_in_property(start);
+}
+
+
+void mark_selectors_in_property(chunk_t *start)
+{
+   chunk_t *tmp = chunk_get_next_type(start, CT_PAREN_OPEN, start->level);
+
+   while (tmp->type != CT_PAREN_CLOSE)
+   {
+      if (tmp->type == CT_WORD &&
+          (memcmp(tmp->str.c_str(), "setter", 6) == 0 ||
+           memcmp(tmp->str.c_str(), "getter", 6) == 0)
+          && tmp->next->type == CT_ASSIGN)
```

Comment:
I am not entirely sure, but this looks like it could need a nullptr check for next.
Does obj-C allow comments/newlines in between the setter and the assign ? Something like `setter /* blub */ =`?